FLEDGE.md

```diff
@@ -1148,6 +1151,113 @@ Currency checking after `scoreAd()` happens only inside component auctions.  If
+
+**Views** don’t require any user interaction and can be registered via `<img>` or `<script>` tags using the `attributionsrc` attribute:
+
+```
+<img src="https://advertiser.example/pixel"
+     attributionsrc="https://adtech.example/register-view?metadata=...">
+
+<script src="https://advertiser.example/register-view"
+        attributionsrc="https://adtech.example/register-view?metadata=...">
+```
+
+**Clicks** can be registered on HTML anchor tags:
+
+```
+<a href="https://advertiser.example/landing"
+   attributionsrc="https://adtech.example/register-click?metadata=...">
+  click me
+</a>
+```
+
+**Clicks** can also be registered via calls to window.open that occur with [transient activation](https://html.spec.whatwg.org/multipage/interaction.html#transient-activation):
+
+```
+// Encode the attributionsrc URL in case it contains special characters,
+// such as '=', that will cause the parameter to be improperly parsed.
+window.open(
+    "https://advertiser.example/landing", "_blank",
+    attributionsrc=encodeURIComponent(
+        "https://adtech.example/register-click?metadata=..."));
+```
+
+When `attributionsrc` is used:
+ * The browser will always look for view and click data on the response to the request that goes to a clickthrough URL ("href" on an `<a>` tag) or impression URL ("src" on an `<img>` or `<script>` tag).
+ * When `attributionsrc` is non-empty, the browser will also send a background request to the URL specified in the value of `attributionsrc`, and then look for view and click data on the response.
+ * The `attributionsrc` HTML attribute may include multiple space-delimited URLs, in which case the browser will issue a background request for each.
+
+
+When the browser sends these request(s), it will include a new `Ad-Auction-Registration-Eligible` header whose value will be exactly one of "view" or "click".
```

Comment:
"exactly one of" here is a bit over-specified, since when combined with greasing the serialized header may not contain those exact strings. In fact, we should probably call out that this is a structured header, and which kind it is.